test/ossl_store_test.c

```diff
@@ -131,6 +138,7 @@ static int test_store_get_params(int idx)
 
     return 1;
 }
+#include <unistd.h>
```

Comment:
This looks weirdly placed. Why not to have it at the top?

Reply:
Debugging leftover trying to figure out why the file wasn't found...